tests/integration/targets/ipam_fixed_address/tasks/main.yml

```diff
@@ -0,0 +1,212 @@
+---
+- module_defaults:
+    group/infoblox.bloxone.all:
+      csp_url: "{{ csp_url }}"
+      api_key: "{{ api_key }}"
+
+  block:
+    # Create a random IP space name to avoid conflicts
+    - ansible.builtin.set_fact:
+        name: "test-ip-space-{{ 999999 | random | string }}"
+        tag_value: "site-{{ 999999 | random | string }}"
+
+    # Basic tests for Subnet
+    - name: "Create an IP space"
+      infoblox.bloxone.ipam_ip_space:
+        name: "{{ name }}"
+        state: "present"
+      register: ip_space
+
+    - name: "Create a Subnet"
+      infoblox.bloxone.ipam_subnet:
+        address: "10.0.0.0/24"
+        space: "{{ ip_space.id }}"
+        state: "present"
+      register: subnet
+
+    - name: "Create a Fixed Address"
+      infoblox.bloxone.ipam_fixed_address:
+        address: "10.0.0.1"
+        name: "test_fixed_address_ansible"
+        match_type: "mac"
+        match_value: "00:00:00:00:00:00"
+        ip_space: "{{ ip_space.id }}"
+        state: "present"
+      register: fixed_address
+
```

Comment:
As per trend for previous objects , we add the tasks for creation/updation , get and assert under one block of code without any empty lines in between . This way empty lines can be used to indicate the next create/update ,read and assert code block